trace = '''Traceback (most recent call last):
  File "/path/to/file", line 23, in funcname
    yield
  File "/path/to/file", line 23, in funcname
    yield
RuntimeError: error

Cause:
  File "/path/to/file", line 23, in funcname
    yield
  File "/path/to/file", line 23
    wrong =
           ^
SyntaxError: error

trailing garbage
